s3parq/publish_parq.py

```diff
@@ -459,7 +459,7 @@ def log_size_estimate(num_bytes):
         yield {'lower': lower, 'upper': upper}
 
 
-def publish(bucket: str, key: str, partitions: List[str], dataframe: pd.DataFrame, redshift_params: dict = None) -> List[str]:
+def publish(bucket: str, key: str, partitions: List[str], dataframe: pd.DataFrame, redshift_params: dict = None, read_access_user: str =None) -> List[str]:
```

Comment:
Can `read_access_user` be moved into `redshift_params` dict on the publish methods since its only part of redshift and not general parquet/S3? That way could keep consistency